neo/Consensus/ConsensusService.cs

```diff
@@ -537,6 +520,16 @@ private void SendPrepareRequest()
             }
             ChangeTimer(TimeSpan.FromSeconds(Blockchain.SecondsPerBlock << (context.ViewNumber + 1)));
         }
+
+        private bool VerifyRequest()
+        {
+            if (!Blockchain.GetConsensusAddress(context.Snapshot.GetValidators(context.Transactions.Values).ToArray()).Equals(context.NextConsensus))
+                return false;
+            Transaction minerTx = context.Transactions.Values.FirstOrDefault(p => p.Type == TransactionType.MinerTransaction);
```

Comment:
We should ensure that is the first TX

Reply:
https://github.com/neo-project/neo/blob/eaeb0a5b01f9a5c37d533125f21081f094f15975/neo/Consensus/PrepareRequest.cs#L39-L40

It is checked here.